src/main/kotlin/skytils/skytilsmod/listeners/ChatListener.kt

```diff
@@ -312,24 +312,20 @@ class ChatListener {
             }
             if (activePet != null) {
                 var rarity = "§f"
-                if (activePet.tier.toString().uppercase() == "UNCOMMON") {
-                    rarity = "§a"
-                } else if (activePet.tier.toString().uppercase() == "RARE") {
-                    rarity = "§9"
-                } else if (activePet.tier.toString().uppercase() == "EPIC") {
-                    rarity = "§5"
-                } else if (activePet.tier.toString().uppercase() == "LEGENDARY") {
-                    rarity = "§6"
-                } else if (activePet.tier.toString().uppercase() == "MYTHIC") {
-                    rarity = "§d"
+                when(activePet.tier.toString().uppercase()) {
+                    ItemRarity.UNCOMMON.rarityName -> rarity = "§a"
+                    ItemRarity.RARE.rarityName -> rarity = "§9"
+                    ItemRarity.EPIC.rarityName -> rarity = "§5"
+                    ItemRarity.LEGENDARY.rarityName -> rarity = "§6"
+                    ItemRarity.MYTHIC.rarityName -> rarity = "§d"
                 }
```

Comment:
Item rarities should have corresponding chat colors, instead maybe use `Enum.valueOf()`

Reply:
You can use .baseColor, has already been fixed just not pr'ed